h2o-core/src/main/java/water/fvec/Frame.java

```diff
@@ -1665,7 +1676,20 @@ public int getCurrentRowSize() throws IOException {
       return StringUtils.bytesOf(sb);
     }
 
-    @Override public int available() throws IOException {
+    static final Pattern doubleQuotePattern = Pattern.compile("\"{1}");
```

Comment:
{1} is unnecessary

Reply:
Yes of course, did that for readability (an escaped quote in two quotes). Removed `{1}`.